index.js

```diff
@@ -2,11 +2,15 @@
 var path = require("path")
 var caller = require('caller');
 
-module.exports = function (toLoad, mocks) {
+var requireInject = module.exports = function (toLoad, mocks, force) {
```

Comment:
I think I'd prefer the export to just take `toLoad` and `mocks` and have the one with the flag be internal only. This would ensure no one will end up depending on using the flag.